runtime.md

```diff
@@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
 See the specification of the `config.json` file for the definition of these fields.
 The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
 In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
-
-## Hooks
-
-Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
```

Comment:
On Fri, Apr 22, 2016 at 10:59:59AM -0700, Vish Kannan wrote:
“What about post stop? If we don't pin the namespaces, I cannot
extract files from the mount namespaces after the init process dies,
for example.”

I'm with @julz [1] and @duglin [2] on this.  Sometimes folks will want
to pin namespaces, and they can do that via sub-containers [1], or
bind-mounts, etc. as they see fit.  But splitting stop/delete is
*requiring* the runtime to pin namespaces, and that seems overly
complicated to me.

[1]: https://github.com/opencontainers/runtime-spec/pull/384/files/5813d96a74b5e753851fdad11ce4731ac7be0576#r60785563
[2]: https://github.com/opencontainers/runtime-spec/pull/384/files/5813d96a74b5e753851fdad11ce4731ac7be0576#r60788563